README.md

```diff
@@ -22,7 +22,7 @@ jobs:
       with:
         fetch-depth: 11 # This value must be set if the lookback configuration option is not disabled for find-code-references. Read more: https://github.com/launchdarkly/ld-find-code-refs#searching-for-unused-flags-extinctions
     - name: LaunchDarkly Code References
-      uses: launchdarkly/find-code-references@v9
+      uses: launchdarkly/find-code-references@v10
```

Comment:
Is there a reason why this version number is inconsistent with the `ld-find-code-refs-github-action` repo release version? i.e. Should this be `v2.2.2` instead of `v10`?

Reply:
When we initially set this up the GitHub team recommended that we use this versioning scheme, however, it doesn't seem to be required. I see https://github.com/marketplace/actions/jira-issue-transition for example is at `v2.0.2`.

Reply:
Ok I'll keep this as is for now but will discuss with Dan O'Brien in the near future to change this to be consistent with our semver versions. Are you able to approve @bwoskow-ld ?